styles/themes/base/components/_input.scss

```diff
@@ -0,0 +1,66 @@
+  // Theme
+  // closeIcon:                'ld-input-clear-icon',
```

Comment:
Do we need that? It will be quite difficult to maintain in all the places)

Reply:
We'll clean it after completing migration to base css and theme props review, it is very convenient now